### DISCUSSION TITLE
HIP-920: NFT Music Metadata Standards

### DISCUSSION THREAD

**Discussion by itsbrandondev** [hiero-ledger/hiero-improvement-proposals#921] [Edits]
The [Hashgraph NFT Working Group](https://twitter.com/hbarnftwg) has submitted a new draft PR https://github.com/hashgraph/hedera-improvement-proposal/pull/920 to create hip-0000-nft-music-metadata.md

The purpose of this HIP is to extend HIP-412 to establish a music metadata standard. This new HIP enables music applications and musicians to fully leverage NFTs on Hedera and for the music industry to easily integrate their existing catalogs. 8 months of collaborative community work has allowed the group to enter the HIP to the draft stage.

[Read the draft HIP](https://deploy-preview-920--hedera-hips.netlify.app/hip-920)

**Comment by angelorc** [comment]
Hello everyone, it's great to see a working group focused on music NFT metadata. My name is Angelo, and I'm the founder of BitSong. I've been working on a metadata standard for years, and the first version was published here: https://github.com/bitsongofficial/bitsongjs/blob/telescope/packages/metadata/test/track.spec.ts#L4. This version is also compatible with OpenSea's ERC721 standard.

I'd love to collaborate with the working group - is there a place for me?

**Reply by itsbrandondev** [reply]
hey! yes we would love to have you as a part of our group : )

feel free to DM us here: https://twitter.com/hbarnftwg

**Comment by mgh14** [comment]
Just a heads up, [this](https://hashscan.io/mainnet/token/0.0.2997600) token is referenced in the hip, and the [ipfs metadata](ipfs://Qmdmy5cNsDm14bSjBAn3FwBeK1QRYb7RT57hLxyapDZZHG) for said token's NFTs has something slightly different for the child items of `credits`. The hip specifies that `producer`, `mix`, `composer`, etc. are string values; however the ipfs metadata has an array of strings for some of these properties, e.g. `"producer": ["Joshua Doerksen", "Brandon Davenport"]`. Not sure if that's something that needs to be called out in the hip or not
